src/account/account/account.cairo

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.8.0 (account/account/account.cairo)
```

Comment:
maybe it's a bit late now, but this `account/account/account.cairo` is very awkward and usually a sign of something not properly abstracted. have we considered merging the account directories into a single one? same with `interface.cairo`

Reply:
Fixed, let me know what you think.

Reply:
looks good!